cmd/machine-config-daemon/start.go

```diff
@@ -154,10 +155,18 @@ func runStartCmd(cmd *cobra.Command, args []string) {
 	}
 
 	if startOpts.onceFrom == "" {
-		err = dn.CheckStateOnBoot()
-		if err != nil {
-			dn.EnterDegradedState(errors.Wrapf(err, "Checking initial state"))
+		for {
```

Comment:
this and the other are just busy loops now, maybe we need to delay each execution